Engine/Results/BacktestingResultHandler.cs

```diff
@@ -642,6 +642,12 @@ public void RuntimeError(string message, string stacktrace = "")
         /// <param name="value">Value for the chart sample.</param>
         public void Sample(string chartName, string seriesName, int seriesIndex, SeriesType seriesType, DateTime time, decimal value, string unit = "$")
         {
+            // Sampling during warming up period skews statistics
+            if (_algorithm.IsWarmingUp)
```

Comment:
this change didn't impact the regression statistics? If not, then I question the assumption that sampling during warm up skews statistics.

Reply:
It didn't. 
We have three types of algorithm with warm up:
1. Those that are not regression algorithm;
2. Those that do not have trades;
3. Those that warm up period is less than one day.
So the regression tests didn't catch the change.

Reply:
Let's add a regression test whose statistics change because of this PR -- confirm by running it on master and observing different statistics than on this PR's branch.